Northstar.Client/mod/scripts/vscripts/ui/menu_lobby.nut

```diff
@@ -178,6 +178,9 @@ void function InitLobbyMenu()
 	AddMenuFooterOption( menu, BUTTON_B, "#B_BUTTON_BACK", "#BACK" )
 	AddMenuFooterOption( menu, BUTTON_BACK, "#BACK_BUTTON_POSTGAME_REPORT", "#POSTGAME_REPORT", OpenPostGameMenu, IsPostGameMenuValid )
 	AddMenuFooterOption( menu, BUTTON_TRIGGER_RIGHT, "#R_TRIGGER_CHAT", "", null, IsVoiceChatPushToTalk )
+	// Client side progression toggle
+	AddMenuFooterOption( menu, BUTTON_Y, "#Y_BUTTON_ENABLE_PROGRESSION", "#ENABLE_PROGRESSION", EnableProgression, ProgressionIsDisabled )
+	AddMenuFooterOption( menu, BUTTON_Y, "#Y_BUTTON_DISABLE_PROGRESSION", "#DISABLE_PROGRESSION", DisableProgression, ProgressionIsEnabled )
```

Comment:
Wouldn't it make more sense to have a single toggle?

Reply:
A toggle doesn't immediately show you which state you are in. I could change the text of the toggle on change, but thats basically just the same as this